onetl/connection/db_connection/jdbc_connection/connection.py

```diff
@@ -206,6 +206,8 @@ def get_df_schema(
         df = self._query_on_driver(query, read_options)
         log.info("|%s| Schema fetched.", self.__class__.__name__)
 
+        log.info("|%s| Using dialect: %s", self.__class__.__name__, self.jdbc_dialect)
```

Comment:
Let's move dialect name before the query is actually performed. In case of exception being raised due to unsupported column type, dialect name will not be logged